src/zenml/cli/integration.py

```diff
@@ -251,14 +251,19 @@ def install(
         force: Force the installation of the required packages.
         uv: Use uv for package installation (experimental).
     """
-    from zenml.cli.utils import is_uv_installed
+    from zenml.cli.utils import is_pip_installed, is_uv_installed
     from zenml.integrations.registry import integration_registry
 
     if uv and not is_uv_installed():
         error(
             "UV is not installed but the uv flag was passed in. Please install uv or remove the uv flag."
         )
 
+    if not uv and not is_pip_installed():
+        error(
+            "Pip is not installed. Please install pip or use the uv flag for package installation."
```

Comment:
One tiny comment: it would be nice to also have the `(--uv)` explanation part in the error message here similar to the others